src/rest-server/src/models/v2/job-attempt.js

```diff
@@ -54,15 +55,15 @@ const healthCheck = async () => {
     return false;
   } else {
     try {
-      const result = await elasticSearchClient.indices.get({
-        index: 'framework',
-      });
+      const result = await elasticSearchClient.cat.health();
       if (result.statusCode === 200) {
         return true;
```

Comment:
Do we also need to check "status" field in the body?

Reply:
how to process with different status